x/emissions/keeper/inference_synthesis/network_inference_builder_test.go

```diff
@@ -128,6 +128,25 @@ func (s *InferenceSynthesisTestSuite) SetupTest() {
 	for _, addr := range addrsStr {
 		s.emissionsKeeper.AddWhitelistAdmin(ctx, addr)
 	}
+
+	err := s.emissionsKeeper.SetTopic(s.ctx, 1, emissionstypes.Topic{
+		Id:              1,
+		Creator:         "creator",
+		Metadata:        "metadata",
+		LossLogic:       "losslogic",
+		LossMethod:      "lossmethod",
+		InferenceLogic:  "inferencelogic",
+		InferenceMethod: "inferencemethod",
+		EpochLastEnded:  0,
+		EpochLength:     100,
+		GroundTruthLag:  10,
+		DefaultArg:      "defaultarg",
+		PNorm:           alloraMath.NewDecFromInt64(3),
+		AlphaRegret:     alloraMath.MustNewDecFromString("0.1"),
+		AllowNegative:   false,
+		InitialRegret:   alloraMath.MustNewDecFromString("0.0001"),
```

Comment:
New param? 
Does it affect also create-topic? 
Then changes on `tests/integration` and stress, etc may be needed?

Reply:
yeah ok given all of this the "initialRegret" probalby shouldn't live on the topic like this especially since it's hardcoded to be zero

Reply:
It's automatically set to zero